File: python_POO/CursoPOOUber/Java/Driver.java
class Driver extends Account {
    Integer id;
    String email;
    String password;
    
    public Driver (Integer id, String name, String document, String email, String password){
        super(name, document);
        this.id = id;
        this.email = email;
        this.password = password;
    }
}

File: python_POO/CursoPOOUber/PHP/Driver.php
<?php
require_once("Account.php");

    class Driver extends Account {
        public $id;
        public $email;
        public $password;

        public function __construct($id, $name, $document, $email, $password){
            parent::__construct($name, $document);
            $this->id = $id;
            $this->email = $email;
            $this->password = $password;
        }
    }
?>